engine/server/engine/centralized_logs/client_implementations/persistent_volume/log_remover/log_remover.go

```diff
@@ -0,0 +1,44 @@
+package log_remover
+
+import (
+	"fmt"
+	"github.com/kurtosis-tech/kurtosis/engine/server/engine/centralized_logs/client_implementations/persistent_volume/logs_clock"
+	"github.com/kurtosis-tech/kurtosis/engine/server/engine/centralized_logs/client_implementations/persistent_volume/volume_consts"
+	"github.com/kurtosis-tech/kurtosis/engine/server/engine/centralized_logs/client_implementations/persistent_volume/volume_filesystem"
+	"github.com/sirupsen/logrus"
+	"strconv"
+	"time"
+)
+
+const (
+	oneWeekDuration = 7 * 24 * time.Hour
+)
+
+// LogRemover removes logs one week older than the log retention period.
+type LogRemover struct {
+	filesystem volume_filesystem.VolumeFilesystem
+
+	time logs_clock.LogsClock
+}
+
+func NewLogRemover(filesystem volume_filesystem.VolumeFilesystem, time logs_clock.LogsClock) *LogRemover {
+	return &LogRemover{
+		filesystem: filesystem,
+		time:       time,
+	}
+}
+
+// Run implements the Job cron interface. It removes logs a week older than the log retention period.
+func (remover LogRemover) Run() {
+	// [LogRetentionPeriodInWeeks] weeks plus an extra week of logs are retained so remove logs a week past that, hence +1
+	numWeeksBack := volume_consts.LogRetentionPeriodInWeeks + 1
+
+	// compute the next oldest week
+	_, weekToRemove := remover.time.Now().Add(time.Duration(-numWeeksBack) * oneWeekDuration).ISOWeek()
```

Comment:
I believe this should work:
```
.Add(-numWeeksBack * oneWeekDuration)
```

`time.Duration(-numWeeksBack)` is actually super confusing when you know what a duration is in Golang (i.e. just an nanosecond count), b/c this literally represents 4 nanoseconds

Reply:
it actually doesn't work weirdly due to `Cannot use '-numWeeksBack * oneWeek' (type int) as the type Duration` - even though `7 * 24 * time.Hour` works...

Reply:
FWIW, casting to time.Duration should still work though, [`time.Duration`](https://pkg.go.dev/time#Duration) is just an int64 underlying so casting `numWeeksBack` to a duration, has still has the effect of multiplying `oneWeekDuration` by 4 even if it technically represents 4 nanoseconds

Reply:
This is weird... This does seem to work here: https://go.dev/play/p/tKH0tLEQFM8?v=goprev
Anyway, not the end of the world...

Reply:
so `numWeeksBack` is get cast int so there's a mismatch when doing `.Add(-numWeeksBack * oneWeek)` but `-4` is an untyped constant that gets cast to a Duration when `.Add(-4 * oneWeek) is evaluated.